### PR TITLE
SKA-10: Create a working endpoint that uses the sunrise / sunset API

### PR DESCRIPTION
**Pull request by thereisnoaddress**
added sunrise sunset API

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -6,3 +6,7 @@
 @app.get('/healthz')
 def healthz():
     return 'OK'
+
+@app.get('/sunrise')
+def sunrise():
+    return '06:00'
```
Comment (thereisnoaddress) [inline main.py:12]: this is not when the sun rises you dumbass

Comment (thereisnoaddress) [reply main.py:12]: that's what copilot says!

**Comment by rohan-sahgal** [comment]
Please avoid using that amount of profanity in commit messages in the future.

**Comment by rohan-sahgal** [comment]
More profanity would be preferable

File: main.py (hunk 1 above)
Comment (rohan-sahgal) [reply main.py:12]: hahaha copilot is a dumbass

Comment (rohan-sahgal) [reply main.py:12]: looks good as a placeholder besides the dumbassery

Comment (rohan-sahgal) [inline main.py:12]: Sunset at 6:00?? What is this, autumn?

**Review by rohan-sahgal — APPROVED** [review]
I LOOOOOOOOOOOOVE this PULL REQUEST

**Comment by thereisnoaddress** [comment]
> I LOOOOOOOOOOOOVE this PULL REQUEST

Alt Text

**Pull request merged**